drivers/cc2420/cc2420.c

```diff
@@ -87,7 +87,17 @@ int cc2420_init(cc2420_t *dev)
     cc2420_reg_write(dev, CC2420_REG_MDMCTRL0, reg);
 
     /* go into RX state */
-    cc2420_set_state(dev, CC2420_GOTO_RX);
+    /* this doesn't compile, compiler complains that the enum type of
+     * CC2420_GOTO_RX cannot be converted to netopt_t as required by the
+     * function.
+     * The numerical value of CC2420_GOTO_RX would be NETOPT_ADDRESS (2), which
+     * can't be right.
+     * cc2420_set_state(dev, NETOPT_STATE_RX) in turn is not implemented
+     * (returns -ENOTSUP), so this was a noop for a while.
+     * TODO: revisit and confirm.
+     * (Kaspar 07/2020)
+     */
+    //cc2420_set_state(dev, CC2420_GOTO_RX);
```

Comment:
Maybe split this out? Someone familiar with that hardware could have a look independently of this PR. I would also be willing to ACK such a PR after I can confirm with code review that this was indeed a noop previously.

Reply:
done: https://github.com/RIOT-OS/RIOT/pull/14694

Reply:
but the gcc that's used in this PR (10.1.0 vs the old 4.x from TI) won't compile as is, so #14694 is a dependency.